devops/openshift/init-secrets.yml

```diff
@@ -206,3 +208,15 @@ parameters:
     required: true
     description: |
       Zeebe authorization server url.
+  - name: QUEUE_DASHBOARD_USER
+    description: Queue dashboard user
+    displayName: User for queue monitoring dashboard
```

Comment:
may you include the name `bull board` too

Reply:
Didn't use bullboard to avoid using a 3rd party framework/product name